server/src/main/java/io/druid/client/cache/MemcachedCache.java

```diff
@@ -273,15 +283,20 @@ public void addMeter(String name)
         @Override
         public void removeMeter(String name)
         {
```

Comment:
Correct. Removes are ignored. I prevent memory leaks by limiting the set of meters allowed. to isInteresting